cruntime does not exist

I got the following error when doing `cmake ..` with the latest commit:
```bash
-- Performing Test HAVE_STD_REGEX -- success
-- Performing Test HAVE_GNU_POSIX_REGEX -- failed to compile
-- Performing Test HAVE_POSIX_REGEX -- success
-- Performing Test HAVE_STEADY_CLOCK -- success
-- Found PythonInterp: /gsa/jpngsa/home/t/u/tungld/dl/anaconda2/envs/python3.6/bin/python (found version "3.6.9")
-- pybind11 v2.4.dev4
-- INCLUDE_ONNX_ML Dialect TRUE
CMake Error at src/CMakeLists.txt:69 (install):
  install TARGETS given target "cruntime" which does not exist in this
  directory.
```

Any suggestion to resolve this error?

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -65,5 +65,3 @@ target_include_directories(onnx-mlir PRIVATE ${CMAKE_BINARY_DIR})
 target_include_directories(onnx-mlir PRIVATE ${ONNX_MLIR_BIN_ROOT})
 
 install(TARGETS onnx-mlir DESTINATION bin)
-install(FILES Runtime/DynMemRef.h DESTINATION include)
-install(TARGETS cruntime DESTINATION lib)

--- a/src/Runtime/CMakeLists.txt
+++ b/src/Runtime/CMakeLists.txt
@@ -18,3 +18,5 @@ target_include_directories(pyruntime
         PRIVATE ${ONNX_MLIR_SRC_ROOT} ${ONNX_MLIR_BIN_ROOT}
         ${ONNX_MLIR_SRC_ROOT})
 add_dependencies(pyruntime cruntime)
+install(FILES Runtime/DynMemRef.h DESTINATION include)
+install(TARGETS cruntime DESTINATION lib)

